apps/app/hooks/useWindow.tsx

```diff
@@ -42,5 +45,7 @@ export const WindowProvider: FunctionComponent<WindowProviderProps> = ({ childre
     };
   }, []);
 
-  return <WindowContext.Provider value={{ breakpoint, scroll }}>{children}</WindowContext.Provider>;
+  return (
+    <WindowContext.Provider value={{ windowWidth, scroll }}>{children}</WindowContext.Provider>
+  );
```

Comment:
Not sure whats the difference here other than the rename.

Reply:
Ah yes its just a rename, I forgot I decided to use the original implementation